File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/avanca_papel.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class AvancaPapel extends TermicaCommand {
  final int _linhas;

  AvancaPapel(this._linhas) : super('AvancaPapel');

  @override
  functionParameters() => '"linhas"' ':' '$_linhas';
}

File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/AvancaPapel.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class AvancaPapel(private val linhas: Int) : TermicaCommand("AvancaPapel") {
    override fun functionParameters(): String {
        return "\"linhas\"" + ":" + linhas
    }
}